packages/theming/win32-theme/src/createFontAliasTokens.ts

```diff
@@ -0,0 +1,11 @@
+import { getOfficeAliasTokens } from './getOfficeAliasTokens';
+import { Variants } from '@fluentui-react-native/theme-types';
+import { memoize } from '@fluentui-react-native/memo-cache';
+import { mapFontPipelineToTheme } from '@fluentui-react-native/theming-utils';
+
+function createFontAliasTokensWorker(officeTheme: string): Partial<Variants> {
```

Comment:
>officeTheme: string [](http://example.com/codeflow?start=37&length=19)

Would the font tokens change between themes? Maybe we don't take a theme as input, and just hardcode fetching one specific alias token file here?